docs/src/tutorial/kubectl.md

```diff
@@ -0,0 +1,135 @@
+# Basic operations with Kubernetes using kubectl
+
+## What you will need
+
+Before you begin, make sure you have the following:
+
+- A bootstrapped Canonical Kubernetes cluster
+(See [Getting Started](https://github.com/canonical/k8s-snap/blob/main/docs/src/tutorial/getting-started.md))
+
```

Comment:
I think we should state the assumption that the user doesn't already have another version of  kubectl installed
- You are using the built-in `kubectl` command from the snap